lib/components/settings/widgets/settings_card.dart

```diff
@@ -132,13 +145,14 @@ class _SettingsCardState extends State<SettingsCard> {
                       setState(
                         () {
                           if (widget.type ==
-                                  SettingsElementModelType.EXPANDABLESWITCH ||
+                                  SettingsElementModelType.expandable_switch ||
```

Comment:
having trouble reading and understanding this code myself, cant imagine others doing so; consider declaring a variable for this boolean expression with an appropriate name